Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics.MicroGraphics/Driver/MicroGraphics.cs

```diff
@@ -77,6 +78,21 @@ public IFont CurrentFont
         /// </summary>
         public TextDisplayConfig DisplayConfig { get; private set; } = new TextDisplayConfig();
 
+        /// <summary>
+        /// Optional enforced deplay between updates when calling ShowBuffered
+        /// </summary>
+        public TimeSpan DelayBetweenFrames { get; set; } = TimeSpan.Zero;
+
+        object _lock = new object();
+
+        bool isUpdating = false;
+        bool isUpdateRequested = false;
+
+        /// <summary>
+        /// Time of last display update when callng ShowBuffered
+        /// </summary>
+        DateTime lastUpdated;
```

Comment:
is this supposed to be public or protected? I ask because it has a /// <summary>

Reply:
intentionally private but I thought the comment was helpful